code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -265,6 +267,9 @@
 	else
 		return FALSE
 
+/obj/structure/closet/proc/weld_act(weld_state)

```

Comment:
mind naming this something else like `after_weld` or something so someone doesn't read this and confuse it with `welder_act`?